packages/shared/src/lib/featureManagement.ts

```diff
@@ -93,3 +93,8 @@ export const briefCardFeedFeature = new Feature(
 );
 
 export const briefUIFeature = new Feature('brief_ui', isDevelopment);
+
+export const showSquadUnreadPosts = new Feature(
+  'show_squad_unread_posts',
+  isDevelopment,
```

Comment:
This should change?

Reply:
`isDevelopment`? Just did as the previous ones, that sets `isDevelopment` as default value. It will default to false in production anyways.